net-p2p/ipfs-desktop-bin/ipfs-desktop-bin-0.39.0.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit desktop xdg unpacker optfeature
+DESCRIPTION="A desktop client for IPFS. The IPFS's Native Application"
+HOMEPAGE="https://github.com/ipfs-shipyard/ipfs-desktop"
+SRC_URI="https://github.com/ipfs/ipfs-desktop/releases/download/v${PV}/ipfs-desktop-${PV}-linux-amd64.deb"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND="
+x11-libs/gtk+:3
+x11-libs/libnotify
+dev-libs/nss
+x11-libs/libXScrnSaver
+x11-libs/libXtst
+x11-misc/xdg-utils
+app-accessibility/at-spi2-core
+sys-apps/util-linux
+app-crypt/libsecret
+"
+RDEPEND="${DEPEND}"
+S="${WORKDIR}"
+
+src_prepare(){
+default
+	unpacker "${S}/usr/share/doc/ipfs-desktop/changelog.gz"
+	# to prevent ipfs-desktop install build-in ipfs exe into /usr/local/bin or ${HOME}/.local/bin
+	# sed -i "s/^binpaths=\(.*\)$/binpaths=\'\/usr\/bin\'/" "${S}/opt/IPFS Desktop/resources/app.asar.unpacked/node_modules/kubo/kubo/install.sh"
```

Comment:
What is this? Is there a reason to leave a comment?

Reply:
It's for experimental... I'm trying to make ipfs-desktop run with system-wide ipfs, not the bundled...But It doesn't work.

Reply:
Which was the problem?

Reply:
You need to remove `resources/app.asar.unpacked/node_modules/kubo/kubo/ipfs` and then adjust `resources/app.asar.unpacked/node_modules/kubo/src/index.js` to grab ipfs binary from PATH.